cdap-docs/admin-manual/source/security/authorization.rst

```diff
@@ -79,36 +67,42 @@ be made finer-grained in the future.)
+  create the entity. For example, creating a namespace requires *ADMIN* on the namespace itself.
 - A **read** operation (such as reading from a dataset or a stream) on an entity requires
-  *READ* on the entity **and** *READ* on the parent.
+  *READ* on the entity.
 - A **write** operation (such as writing to a dataset or a stream) on an entity requires
   *WRITE* on the entity.
 - An **admin** operation (such as setting properties) on an entity requires *ADMIN* on
   the entity.
-- A **delete** operation on an entity requires *ADMIN* on the entity.
-- A **list** operation (such as listing or searching applications, datasets, streams,
+- A **delete** operation on an entity requires *ADMIN* on the entity. Note that if the deletion operation will delete
+  multiple entities, *ADMIN* is required on all the entities. For example, delete on a namespace requires *ADMIN* on
+  all entities in the namespace.
+- A **execute** operation on a program requires *EXECUTE* on the program.
+- A **list** or **view** operation (such as listing or searching applications, datasets, streams,
   artifacts) only returns those entities that the logged-in user has at least one (*READ*,
-  *WRITE*, *EXECUTE*, *ADMIN*) privilege on.
-- A **view** operation on an entity only succeeds if the user has at least one (*READ*,
-  *WRITE*, *EXECUTE*, *ADMIN*) privilege on it.
+  *WRITE*, *EXECUTE*, *ADMIN*) privilege on or on any of its descendants.
+- A **get** operation on an entity (such as getting the dataset property, app detail) only succeeds if the user has
+  at least one (*READ*, *WRITE*, *EXECUTE*, *ADMIN*) privilege on it.
+- Only admins of the authorization backend can grant or revoke the privileges.
 
 Additionally:
 
-- Upon successful creation of an entity, the logged-in user receives all privileges
-  *(READ*, *WRITE*, *EXECUTE*, and *ADMIN)* on that entity.
-- Upon successful deletion of an entity, all privileges on that entity for all users are revoked.
-
-CDAP supports hierarchical authorization enforcement, which means that an operation that
-requires a certain privilege on an entity is allowed if the user has the same privilege on
-the entity's parent. For example, reading from a CDAP dataset will succeed even if the
-user does not have specific *READ* privileges on the dataset, but instead has *READ*
-privileges on the namespace in which the dataset exists.
-
-Note: :ref:`CDAP Instance <security-cdap-instance>` is not a part of the authorization hierarchy. For example, READ on a CDAP Instance does not give READ on any of the Namespaces or any other entities in the Namespace.
+- Upon successful creation/deletion of an entity, the privileges remain unaffected.
+- More privileges are needed for the following operations in the following cases:
+    - Deploying app needs *READ, WRITE, EXECUTE,* or *ADMIN* on the artifact, *ADMIN* privilege on all the datasets and
```

Comment:
Yes. If the dataset/stream exists, we will call update instead of create, which still requires ADMIN privilege. Will update the case with app deployment with new artifact.